pipeline/inputs/windows-exporter-metrics.md

```diff
@@ -21,6 +21,9 @@ The initial release of Windows Exporter Metrics contains a single collector avai
 | we.logical\_disk.allow\_disk\_regex | Specify the regex for logical disk metrics to allow collection of. Collect all by default. | "/.+/"    |
 | we.logical\_disk.deny\_disk\_regex  | Specify the regex for logical disk metrics to prevent collection of/ignore. Allow all by default. | `NULL`    |
 |we.net.allow\_nic\_regex            | Specify the regex for network metrics captured by the name of the NIC, by default captures all NICs but to exclude adjust the regex.  | "/.+/"    |
+| we.service.where           | Specify the where clause for retrieving service metrics.  | `NULL`   |
+| we.service.include         | Specify the key value condition pairs for includeing condition to construct where clause of service metrics. | `NULL`   |
+| we.service.exclude         | Specify the key value condition pairs for excludeing condition to construct where clause of service metrics. | `NULL`   |
```

Comment:
Typo on `excluding`

Reply:
I think that we should unify the sentence structure for include one.
https://github.com/fluent/fluent-bit-docs/pull/1140/commits/9411809ec955cde5ec48889adedab2c28dba70de